lib/rest.ex

```diff
@@ -1041,6 +1054,33 @@ defmodule Crux.Rest do
     Rest.Base.queue(:delete, Endpoints.guild(guild_id))
   end
 
+  @type audit_log_options ::
+          %{
+            optional(:user_id) => snowflake(),
+            optional(:action_type) => pos_integer(),
+            optional(:before) => snowflake(),
+            optional(:limit) => pos_integer()
+          }
+          | [
+              {:user_id, snowflake()},
```

Comment:
The `,`s at the end of the lines should be `|`s instead, although they seem to display identically in the end.

---
A typedoc linking the function this is used in would be nice.